content/python/concepts/built-in-functions/terms/map/map.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'map()'
+Description: 'Returns an iterator that takes a function and applies it to every item in an iterable.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `map()` built-in function takes a function and applies it to every item in an iterable. The output of `map()` is a map object, in the examples below `list()` is applied to the map expression to return a new list.
```

Comment:
```suggestion
The `map()` built-in function accepts a function and applies it to every item in an iterable. It outputs a map object. In the examples below, the `list()` function is applied to the map expression to return a new list.
```

Reply:
If you choose to use `list()` in the `filter()` entry, can you put a similar explanation like this one mentioned here?

Reply:
In addition, can we move this line to right before the examples? So for this entry, on line 29?

Reply:
Sorry, not clear on what you're suggesting. Do want to move the new (last) sentence in your edit from line 17 to 29?

Reply:
@caupolicandiaz The edits you made work as well! Thanks for incorporating this suggestion 🙏